2022/incoming/let-the-elves-import-your-packages.pod

```diff
@@ -0,0 +1,75 @@
+Author: Bartosz Jarzyna <bbrtj.pro@gmail.com>
+Title: Let the Elves Import Your Packages
+Topic: Use 'all' packages from the namespace at once
+
+=encoding utf8
+
+=head1 Let the Elves Import Your Packages
+
+Same as every year, the last few days before Christmas are very busy for the Elves.
+Those little tinkers are best known for hand-craftng all the presents kids
```

Comment:
Still a typo here, unfortunately. "hand-craftng" should be "hand-crafting".

Reply:
Whoops, I guess I'm blind. Well, it's too late now, isn't it?

Reply:
Resolved in #273